modules/drivers/druid-jdbc/src/metabase/driver/druid_jdbc.clj

```diff
@@ -31,10 +32,13 @@
   (defmethod driver/database-supports? [:druid-jdbc feature] [_driver _feature _db] supported?))
 
 (defmethod sql-jdbc.conn/connection-details->spec :druid-jdbc
-  [_driver {:keys [host port] :as _db-details}]
+  [_driver {:keys [host port auth-enabled auth-username] :as _db-details}]
```

Comment:
It is common practice to use leading underscore in names that are not used later. As the `_db-details` are in use now, could you please remove it?